patches/net/minecraft/world/gen/feature/structure/DesertPyramidStructure.java.patch

```diff
@@ -0,0 +1,35 @@
+--- a/net/minecraft/world/gen/feature/structure/DesertPyramidStructure.java
++++ b/net/minecraft/world/gen/feature/structure/DesertPyramidStructure.java
+@@ -1,5 +1,7 @@
+ package net.minecraft.world.gen.feature.structure;
+ 
++import com.google.common.collect.Lists;
++import net.minecraft.entity.EntityType;
+ import net.minecraft.init.Biomes;
+ import net.minecraft.util.SharedSeedRandom;
+ import net.minecraft.util.math.BlockPos;
+@@ -7,8 +9,12 @@
+ import net.minecraft.world.biome.Biome;
+ import net.minecraft.world.gen.IChunkGenerator;
+ 
++import java.util.List;
++
+ public class DesertPyramidStructure extends ScatteredStructure<DesertPyramidConfig>
+ {
++    private static final List<Biome.SpawnListEntry> spawnList = Lists.newArrayList(new Biome.SpawnListEntry(EntityType.HUSK, 1, 1, 1));
++
+     protected String getStructureName()
+     {
+         return "Desert_Pyramid";
+@@ -25,6 +31,11 @@
+         return new DesertPyramidStructure.Start(worldIn, random, x, z, biome);
+     }
+ 
++    public List<Biome.SpawnListEntry> getSpawnList()
```

Comment:
Do you mind adding @Override decorator. I found it helps greatly porting in the future in case the name has changed.

Reply:
Ohh sorry forgot it. 
Sure will add it